### ISSUE TITLE
Provide tk installation with libXft support in Python installations

### ISSUE THREAD

**Issue by FCamborda** [astral-sh/uv#7848]
We're trying to use customtkinter (which depends on tkinter, already delivered with the uv Python installation). As seen in this issue, we export the necessary env. variables to get tkinter to work (https://github.com/astral-sh/uv/issues/7036) 

```shell
uv python install 3.9
uv venv testenv --python 3.9
BASE_PATH="$(dirname $(dirname $(readlink $(which python))))"
export TK_LIBRARY="$BASE_PATH/lib/tk8.6"
export TCL_LIBRARY="$BASE_PATH/lib/tcl8.6"
uv pip install customtkinter
python
```

Then call the following code (taken from https://customtkinter.tomschimansky.com)
```python
import customtkinter

def button_callback():
    print("button clicked")

app = customtkinter.CTk()
app.geometry("400x150")

button = customtkinter.CTkButton(app, text="my button", command=button_callback)
button.pack(padx=20, pady=20)

app.mainloop()
```
The rendering is then quite ugly.  Apparently this is a known issue when tk is built without libXft support:
https://github.com/TomSchimansky/CustomTkinter/issues/2596

Could you confirm that this is the case when installing Python in uv? If so, is there a way to include this to make rendering more pleasent to the eyes?

We are using uv 0.4.9 in native Ubuntu 22.04.4 LTS

Thanks.

**Comment by zanieb** [comment]
@learncodingforweb this sounds like a separate issue. Please see #9452 / open a new issue including the output.. Make sure you're on the latest version of uv and reinstall your managed Python version with `uv python install <version> --reinstall` to ensure you have the latest patches.

**Comment by zanieb** [comment]
@learncodingforweb again, please open a new issue including a reproduction and output as described in #9452. It's bad form to hijack another issue.

**Comment by geofft** [comment]
> Could you confirm that this is the case when installing Python in uv?

Yes, it is the case that our managed Python installs are built without Xft. I guess we ought to do that. I opened astral-sh/python-build-standalone#740, not sure when I'll get to it.
